app/models/facility.rb

```diff
@@ -81,6 +82,7 @@ class Facility < ApplicationRecord
   validates :district, presence: true
   validates :state, presence: true
   validates :country, presence: true
+  validates :zone, presence: true
```

Comment:
This should only be validated on `create`, correct? Otherwise we are going to have invalid facilities in prod that have empty zones.

Reply:
discussed on slack, this is desired so that if admins do edit any facilities, they will be forced to enter a zone.  Facility updates are very rare in production.